scripts/installErizo.sh

```diff
@@ -21,6 +21,7 @@ elif [ "$(expr substr $(uname -s) 1 5)" == "Linux" ]; then
 fi
 
 export ERIZO_HOME=$ROOT/erizo
+export PATH=$HOME/.local/bin:$PATH; conan
```

Comment:
I'm not sure if it is because Ubuntu 20.04 but without it the script wouldn't find conan. For what I have seen, pip doesn't install conan globally and the shell wouldn't find it if I don't specify the path.